### DISCUSSION TITLE
Prism for ios ?

### DISCUSSION THREAD

**Discussion by iamcienlim** [Hash-Studios/Prism#212] [Ideas]
Hi, may I know is that possible we can have prism on ios? Love this wallpaper app so much!

**Comment by codenameakshay** [comment]
Currently, we only support Android due to the fact that we use some native Java code, and we don't exactly know Swift. But if you want you can port it on iOS, as most of the Flutter code will work on it without any issues. But for the wallpaper setting code, you may need to write Swift code for that.
